src/pages/index.js

```diff
@@ -100,30 +256,39 @@ const addCardPopup = new PopupWithForm({
 });
 addCardPopup.setEventListeners();
 
+const editAvatarPopup = new PopupWithForm({
+  popupSelector: "#change__profile_picture",
+  handleFormSubmit: handleAvatarFormSubmit,
+});
+editAvatarPopup.setEventListeners();
+
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
   addCardFormValidator.toggleButtonState();
 });
 
-const popupWithImage = new PopupWithImage({
-  popupSelector: ".preview__modal",
-});
-popupWithImage.setEventListeners();
-
-const profileEditFormValidator = new FormValidator(options, profileEditForm);
-profileEditFormValidator.enableValidation();
-
-const addCardFormValidator = new FormValidator(options, addCardFormElement);
-addCardFormValidator.enableValidation();
-
-const userInfo = new UserInfo({
-  profileTitle,
-  profileDescription,
-});
-
 profileEditButton.addEventListener("click", () => {
   const userData = userInfo.getUserInfo();
   profileTitleInput.value = userData.name;
   profileDescriptionInput.value = userData.job;
   editProfilePopup.open();
 });
+
+profileAvatarEditButton.addEventListener("click", () => {
+  editAvatarPopup.open();
```

Comment:
Please, call `toggleButtonState` for the avatar as well to toggle the button on opening